src/vs/platform/quickinput/browser/media/quickInput.css

```diff
@@ -239,32 +240,15 @@
 	margin-left: 5px;
 }
 
-.quick-input-widget.show-checkboxes .quick-input-list .quick-input-list-rows {
-	margin-left: 10px;
-}
-
-.quick-input-widget .quick-input-list .quick-input-list-checkbox {
-	display: none;
-}
-.quick-input-widget.show-checkboxes .quick-input-list .quick-input-list-checkbox {
-	display: inline;
-}
-.quick-input-widget.show-checkboxes .quick-input-list-entry.not-pickable {
-	margin-left: -10px;
-
-	.quick-input-list-checkbox {
-		display: none;
-	}
-}
-
 .quick-input-list .quick-input-list-rows > .quick-input-list-row {
 	display: flex;
 	align-items: center;
 }
 
 .quick-input-list .quick-input-list-rows > .quick-input-list-row .monaco-icon-label,
 .quick-input-list .quick-input-list-rows > .quick-input-list-row .monaco-icon-label .monaco-icon-label-container > .monaco-icon-name-container {
-	flex: 1; /* make sure the icon label grows within the row */
+	flex: 1;
+	/* make sure the icon label grows within the row */
```

Comment:
what's up with these comments showing up after the properties they're talking about. That's weird, no?

Reply:
Ah, I think that happened when I ran Format Document with the built-in CSS server